worldwind/src/commonMain/kotlin/earth/worldwind/render/program/TriangleShaderProgram.kt

```diff
@@ -61,49 +76,67 @@ open class TriangleShaderProgram : AbstractShaderProgram() {
                     float miterLength = 1.0 / max(dot(miter, normalA), invMiterLengthCutoff);
                     
                     gl_Position = pointBScreen;
-                    gl_Position.xy = gl_Position.xy + (corner * miter * lineWidth * miterLength) / screen.xy;
+                    gl_Position.xy = gl_Position.xy + (corner * miter * useLineWidth * miterLength) / screen.xy;
+                    
+                    /* Transform the vertex tex coord by the tex coord matrix. */
+                    if (enableTexture) {
+                        texCoord = (texCoordMatrix * vec3(vertexTexCoord.x, 0.0, 1.0)).st;
+                    }
                 }
-                
-                /* Transform the vertex tex coord by the tex coord matrix. */
-                if (enableTexture) {
-                    texCoord = (texCoordMatrix * vec3(vertexTexCoord, 1.0)).st;
+                   
+                outVertexColor = enableVertexColorAndWidth ? vertexColor : color;
+                if(enablePickMode && pickIdOffset != 0) {
```

Comment:
What happens when pick mode and pickIdOffset==1?

Reply:
if we have full batch of 256 pathes we just offset blue component of pickColor by 1/255 for all pathes, except last. the last path index gonna overflow blue component and have 1/255 in green component